host/src/patches/MurderScene.cpp

```diff
@@ -75,23 +75,49 @@ LONG WINAPI MurderSceneVEH(PEXCEPTION_POINTERS ptrs)
 		bool found = false;
 
 		{
+			printf("Warning: Exception catched! Trying to prevent...\n");
+
 			std::lock_guard<std::mutex> lock(g_taskListMutex);
 
 			found = (g_runningThreadpoolTasks.find(GetCurrentThreadId()) != g_runningThreadpoolTasks.end());
 		}
-		
+
 		if (found)
 		{
 			// stub out this threadpool task
 			if (((uintptr_t)ptrs->ExceptionRecord->ExceptionAddress >> 36) == 0x7FF)
 			{
+				printf("Info: Loading Exception. Preventing Crash...\n");
+				ptrs->ContextRecord->Rip = (DWORD64)RaiseTrailBlazer;
+				ptrs->ContextRecord->Rsp &= 0xFFFFFFFFFFFFFFF0;
+				ptrs->ContextRecord->Rsp |= 8;
+
+				return EXCEPTION_CONTINUE_EXECUTION;
+			}
+			else {
+				printf("Warning: Unknown Exception! Experimental workaround...\n");
```

Comment:
Does this one fix the 'crash after loading'? It could be base addresses differ on some systems.